### ISSUE TITLE
[A11y] Explore Collections - Grid - Image alt attribute

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1869]
### Operating System

Windows

### Browser

Edge

### Affected page

Explore Collections

### Admin user?

Yes

### Issue Description

Images in grid mode don't have an alt attribute, which is required for all images

### Suggested Fix

Just change these to `alt=""`.  The text below the images is wrapped in the same `<a>` tag, so these should be fine to call decorative.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Verify that no images lack "alt" attributes.  Something like the WAVE toolbar should be able to handle this quickly.
